packages/utils/src/composeAbortSignals.ts

```diff
@@ -10,29 +10,52 @@ export type ComposedAbortSignal = AbortSignal & { destroy: () => void }
  * aforementioned instance of AbortSignal will have more and more listeners added
  * but never cleaned.
  */
-export function composeAbortSignals(...signals: AbortSignal[]): ComposedAbortSignal {
-    if (signals.length === 0) {
-        throw new Error('must provide at least one AbortSignal')
-    }
+export function composeAbortSignals(
+    ...signals: (AbortSignal | undefined)[]
```

Comment:
@harbu, I've change this one to be an array of signals or undefineds, and droppped `null`. It'll match the convention and address one of your feedback comments.

Cheers.